content/release-notes/1.17.0.md

```diff
@@ -0,0 +1,44 @@
+---
+date: 2020-07-13T08:00:00-07:00
+linktitle: "1.17.0"
+title: "1.17.0"
+kubernetes: "1.16, 1.17, and 1.18"
+---
+
+{{<features>}}
+* GitOps - Added Github Enterprise support.
+* Snapshots - Added a "Internal Storage (Default)" option to the providers list for KURL embedded clusters. Which will switch back to the default configuration that velero was set up with when the app was installed.
+* Added `when` field support for config groups.
+{{</features>}}
+
+{{<changes>}}
+* Kotsadm - New Support bundles will be shown at the top of the list instead of the bottom.
+* Kotsadm - Showing warnings on the Version History page in the Admin Console when installing an application with invalid YAML.
+* Kotsadm - Added a page title for the 'View files' and 'Config' tabs.
+* Removed the KOTS metadata that was added to all application manifests. Which included the local application and the version sequence injected as annotations to all deployed manifests.
+* Show the airgap upload progress percentage more granularly.
+* Removed the "Re-analyze" button from the troubleshoot support bundle analysis page.
+* Auto deploy the app's first version when installing the app and no preflight checks are available or preflight checks pass.
+* Update to import k8s 1.18 libraries.
+* Show a descriptive error message when `kubectl kots download` is called with an app slug that does not exist.
+{{</changes>}}
+
+{{<fixes>}}
+* Fixed a bug in KOTS CLI where using `--deploy` flag when checking for updates does not deploy the latest release.
+* Fixed a bug in KOTS CLI where automated installs fail to deploy if no preflight checks are available.
+* Fixed a bug in KOTS CLI where checking for updates shows `There are no application updates available` even when updates are actually available.
+* Fixed an issue where license expiration isn't checked until after fetching application.
+* Fixed a bug where preflight checks were not triggered for airgap or automated installs.
```

Comment:
Does this always happen, or just in a specific situation? If so, what situation? If not, I'd use stronger wording to convey this is seriously broken (and probably put this at the top of the bug fixes list). Also, what do you mean by `automated installs`? Is this an install via `kots install?`

Reply:
This always happens. And yes, by automated installs i mean using `kots install` and providing a license and config values, which should bypass the config and preflights screen, in which case, the api was not triggering the preflights.

Reply:
```suggestion
* Fixed a bug where preflight checks fail to trigger for UI-based airgap installations, or for automated installations where license and configuration has been provided to `kots install` via the CLI. 
```